### PR TITLE
Cairns Operations

### PR DESCRIPTION
**Pull request by mattkelly4**
## Summary
Updates the Cairns procedures to remove outdated information and add additional details & examples for a variety of scenarios.

## Changes
**Fixes**:
- Improves clarity of Cairns Aerodrome and TCU pages

**Additions**:
- Helicopter operations to The Pier
- Operational notes and coordination requirements for visual approaches
- YBCS STAR clearance advice for KEN

**Removals**:
- Trinity Airspace release (now released by default)
- Reciprocal runway operations (overcomplication for VATSIM needs)

### CODE REVIEW

**Review by tylerthetiletiler — DISMISSED** [review]

**Review by alphadelta332 — APPROVED** [review]

**Review by Slep-wt — DISMISSED** [review]

**Comment by mattkelly4** [comment]
Sorry, just need to change A020 to A020V, standby on merging

**Review by glennawatson — CHANGES_REQUESTED** [review]

File: docs/aerodromes/classc/Cairns.md [hunk 1]
```diff
@@ -13,54 +13,49 @@
 | Cairns ATIS        |                | 131.100          | YBCS_ATIS                                |
 
 ## Airspace
-CS ADC is responsible for the Class C Airspace within the CS CTR `SFC` to `A010`.
+CS ADC is responsible for the Class C airspace within the CS CTR `SFC` to `A010`.
+
+!!! important
+    CS ADC is responsible for issuing visual approach clearances to some aircraft tracking for runway 33 in CTA, as well as initial airways clearances for VFR aircraft entering the CTR from class G. See [Runway 33 Arrival Procedures](#runway-33) for more information.
```
[the remark this answers is not in the document]
Comment (mattkelly4) [reply docs/aerodromes/classc/Cairns.md:19]: It's a quick note to indicate that ADC has additional responsibilities. They are outlined in detail by following the link

**Review by alphadelta332 — DISMISSED** [review]

File: docs/aerodromes/classc/Cairns.md [hunk 2]
```diff
@@ -13,54 +13,49 @@
...
+When weather conditions are suitable, ADC shall visually separate aircraft operating in the circuit from aircraft on the extended runway centreline, within 5nm. ADC shall also separate aircraft operating on (or departing to join) the Western VFR Corridor, within the CTR. 
 
-## Runway Selection
-The preferred Runway direction is Runway 15 
+!!! Note 
+    ADC shall advise TCU when this separation cannot be applied.
 
-## Separation
+### Departures in IMC
+When instrument approaches are in use to runway 15, and visual separation is not possible due to the weather conditions (i.e. VMC does not exist below `A030`), do not issue a takeoff clearance to an aircraft if another arriving aircraft is established on less than a 7nm final. This prevents a loss of separation between the two aircraft.
```
[the remark this answers is not in the document]
Comment (mattkelly4) [reply docs/aerodromes/classc/Cairns.md:38]: Sorry Glenn, this suggestion doesn't make a great deal of sense in my head. Might leave it as is.

File: docs/aerodromes/classc/Cairns.md [hunk 3]
```diff
@@ -13,54 +13,49 @@
...
 
-The above does not apply if tower accepts responsibility for separation.
+### Runway 33
+When visual approaches are in use, expect light & medium category aircraft (B737/A320 and below) to track via a straight in visual approach, or track via a visual right base. Aircraft inside CTA tracking for a right base to runway 33 will pass overhead the CTR, so CS TCU **will not** clear them for a visual approach. These aircraft will be coordinated by CS TCU.
 
-!!! Note
-    The ADC may not accept responsibility for separation when a jet aircraft is departing Runway 33.
+ADC must assess any conflict with aircraft in the CTR and clear the inbound aircraft for a visual approach when suitable. Due to the small amount of airspace available, it is vital that ADC maintains separation assurance between inbound aircraft and those operating in the CTR, to avoid a delay in issuing the visual approach clearance.
```
[the remark this answers is not in the document]
Comment (mattkelly4) [reply docs/aerodromes/classc/Cairns.md:47]: Still needs to reference separation *assurance*, not just separation here. I'll reword it to implement your other suggested wording changes.

File: docs/aerodromes/classc/Cairns.md [hunk 4]
```diff
@@ -72,63 +67,91 @@ b) RWY 33, All Jets: EAZEE SID, Radar Transition
...
-### Minimum IMC Levels
-Lowest assignable level in IMC for assigned heading is:
-
-| Runway | Jet | Non-Jet|
-| ------ | -------- | -------|
-| 15     | `A030` | `A020`, between 350° to 030° |
-| 33     | `A040`, between 330° to 009°.  `A030`, all other headings | `A040`, between 330° to 009°.  `A020`, between 010° to 060°.  `A024`, between 061° to 070° |
+## VFR Operations
+### The Pier Helipad
+VFR helicopters transit from the Reef to a floating pontoon helipad on the southeastern shore of the CBD, approximately 2nm from the runway 33 threshold. This helipad is referred to as 'The Pier'.
```
[the remark this answers is not in the document]
Comment (mattkelly4) [reply docs/aerodromes/classc/Cairns.md:74]: Appreciate the input but would prefer to leave this sentence structured as originally committed (avoids confusion around whether the helipad is 2nm from RWY33 or whether the CBD is 2nm from RWY33).

File: docs/aerodromes/classc/Cairns.md [hunk 5]
```diff
@@ -13,54 +13,49 @@
...
-a) assigned a level above and vertically separated from the assigned level of the departure and be at a distance which permits uninterrupted profile descent without loss of radar separation;  
-b) be confined to a square base leg to join not less than a 10 mile final; or  
-c) confined to a procedurally separated clearance  
+## Arrivals
+### Runway 15
+When visual approaches are in use, expect light & medium category aircraft (B737/A320 and below) to track via the Creek Corridor, to join an approx 2nm final. These aircraft will be cleared for a visual approach by **CS TCU**. See the Cairns DAP Noise Abatement Procedures chart for more information.
 
-The above does not apply if tower accepts responsibility for separation.
+### Runway 33
+When visual approaches are in use, expect light & medium category aircraft (B737/A320 and below) to track via a straight in visual approach, or track via a visual right base. Aircraft inside CTA tracking for a right base to runway 33 will pass overhead the CTR, so CS TCU **will not** clear them for a visual approach. These aircraft will be coordinated by CS TCU.
```
[the remark this answers is not in the document]
Comment (mattkelly4) [reply docs/aerodromes/classc/Cairns.md:45]: Shall reword separately.

**Review by alphadelta332 — DISMISSED** [review]

**Review by tylerthetiletiler — DISMISSED** [review]

**Review by alphadelta332 — APPROVED** [review]

**Review by tylerthetiletiler — APPROVED** [review]

**Review by LeGiraffe — APPROVED** [review]
